### DISCUSSION TITLE
mounting dev/loop0 on run/initramfs/ltsp/0/looproot failed: No such device

### DISCUSSION THREAD

**Discussion by fragpic** [ltsp/ltsp#291] [Q&A]
I followed the instructions to setup ltsp to serve a .raw KVM image to clients.
The server side of the setup went well without any issues but my client boot isnt working.
I'm able to get the iPXE boot menu and was able to start the boot process but it fails with this issue 

mounting dev/loop0 on run/initramfs/ltsp/0/looproot failed: No such device

Am I missing something?

![image](https://user-images.githubusercontent.com/9153945/86482493-ad049680-bd6f-11ea-8103-a67ba5826968.png)

**Comment by alkisg** [comment]
Did you install ltsp inside the image? Among others, this will put the necessary squashfs module to the initramfs.

From https://ltsp.org/docs/installation:

> In the virtual machine and chroot cases, you're supposed to install the ltsp package to the image, by adding the LTSP PPA and running apt install ltsp epoptes-client, without specifying any other services.

**Comment by fragpic** [comment]
That did the trick!
I somehow missed it since I thought it was related to the epoptes server which I was not planning to use.

Thank you!
